internal/pkg/template/templates/workloads/partials/cf/workload-container.yml

```diff
@@ -43,8 +43,8 @@
 {{- end}}
 {{- end}} {{/* end if eq .WorkloadType "Load Balanced Web Service"*/}}
 {{- if eq .WorkloadType "Backend Service"}}
-{{- if .ServiceConnect }} # remove when release
-  PortMappings: !If [ExposePort, [{ContainerPort: !Ref ContainerPort, Name: !Ref WorkloadName}], !Ref "AWS::NoValue"]
+{{- if and .ServiceConnect (eq .HTTPTargetContainer.Name .WorkloadName)}} # remove when release
+  PortMappings: !If [ExposePort, [{ContainerPort: !Ref ContainerPort, Name: target}], !Ref "AWS::NoValue"]
```

Comment:
this `Name` field is unique within the task definition across containers right? like it would make no sense to have two `'target'` port mapping in different container definitions?

Reply:
Yeah that's true. They can't have two containers with the same name. and each port mapping should have a unique name.